.vscode/settings.json

```diff
@@ -0,0 +1,48 @@
+{
+	// File settings
+	"files.eol": "\n",
+	"files.exclude": {
+		"**/node_modules": true,
+		".cache": true
+	},
+	"files.associations": {
+		// VS Code treats ".prettierrc" as JSON, by default
+		".prettierrc": "yaml"
+	},
+	// Editor settings
+	"editor.formatOnSave": false,
+	"editor.codeActionsOnSave": {
+		"source.fixAll": true,
+		"source.fixAll.eslint": true,
+		"source.fixAll.tslint": true,
+		"source.fixAll.stylelint": true
+	},
+	"editor.unicodeHighlight.ambiguousCharacters": false,
```

Comment:
what does this one solve?

Reply:
Prevents unicode spoofing backdoor attacks
https://code.visualstudio.com/updates/v1_63#_unicode-highlighting

Reply:
Oh, it should be set to `true` 🤦‍♂️ 

It's true by default, so we can remove it.

Reply:
Removed. Thank you for catching that.

Reply:
😁 thanks!